README.md

```diff
@@ -96,6 +96,7 @@ Translations can then be updated with
 ```bash
 summon -p gopass npm run update:translations
 ```
+The file `secrets.yml` will tell `gopass` which keys to get from the store.
```

Comment:
Actually `secrets.yml` is used by summon not gopass. It does the mapping between the gopass key and the environment variable that should be exported